### ISSUE TITLE
want to get rid of the message "Chrome is controlled by automated software"

### ISSUE THREAD

**Issue by kazurayam** [kazurayam/chromedriverfactory#56]
I want to get rid of the message "Chrome is controlled by automated software", as found

<img width="912" alt="With_the_message" src="https://github.com/kazurayam/chromedriverfactory/assets/4467417/844f259e-7e2d-42b8-aea1-a139919e5d03">

See the following post for how to
https://forum.katalon.com/t/chrome-is-being-controlled-by-automated-test-software-how-to-stop-it/122343

---

See [https://help.applitools.com/hc/en-us/articles/360007189411--Chrome-is-being-controlled-by-automated-test-software-notification 1](https://help.applitools.com/hc/en-us/articles/360007189411--Chrome-is-being-controlled-by-automated-test-software-notification)

In short, we can specify an option to ChromeDriver programatically;
```
ChromeOptions options = new ChromeOptions();
options.setExperimentalOption("excludeSwitches", new String[]{"enable-automation"});
WebDriver driver = new ChromeDriver(options);
```

**Comment by kazurayam** [comment]
v0.7.2 supported `enable-automation` option

**Comment by kazurayam** [comment]
The default configuration should NOT include the enable-automation.

The enable-automation option is one way. You can not bring it off. So it is a bad idea to set it on by default.

**Comment by kazurayam** [comment]
Line#57 of https://github.com/kazurayam/chromedriverfactory/blob/master/src/main/java/com/kazurayam/webdriverfactory/chrome/ChromeDriverFactoryImpl.java

```
	private void prepareDefaultSettings() {
	        ...
        	//this.addChromeOptionsModifier(ChromeOptionsModifiers.enableAutomation());
	}
```
